frontend/src/app/dashboard/page.tsx

```
@@ -436,43 +350,66 @@ export default function DashboardPage() {
+                    <div className="border-t border-gray-200 dark:border-gray-700 mb-4"></div>
+
+                    {/* Action Buttons */}
+                    <div className="flex justify-end gap-2">
+                      <button
+                        onClick={() => handleEditProject(project)}
+                        className="flex items-center gap-2 bg-white dark:bg-gray-800 hover:bg-gray-50 dark:hover:bg-gray-700 text-gray-700 dark:text-gray-300 border border-gray-300 dark:border-gray-600 px-4 py-2 rounded-lg text-sm font-medium transition-all duration-300"
                       >
-                        {project.status === 'in_progress' ? 'Continue Assessment' : project.status === 'completed' ? 'Completed' : 'Start Assessment'}
-                        <ArrowRight className="w-4 h-4" />
-                      </Link>
+                        Edit
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)